style.css

```diff
@@ -0,0 +1,74 @@
+header {
+  background-image: url(assets/header-background-mobile.png);
```

Comment:
- [ ] [OPTIONAL] It will be better to remove background-image because it does not affect your style.